app/views/shared/_spinner.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="spinner hidden">
+  <div>
+    <%= image_tag(asset_url('spinner.gif'),
+        srcset: asset_url('spinner@2x.gif'),
+        height: 144,
+        width: 144,
+        alt: '') %>
+  </div>
+</div>
+<% nonce = content_security_policy_script_nonce %>
+<%= nonced_javascript_tag do %>
+  var nonce="<%= nonce %>";
```

Comment:
Aside: I don't understand what this variable is used for 🤷 

(No change necessary since it existed previously, just caught my attention)

Reply:
good q not sure what we use the nonce for inside the JS, also we can use the `j` helper to escape a value for javascript if we're at all paranoid:

Reply:
removed the nonce variable since it doesn't seem to be used and the spinner works without it